File: eval/fixtures/legacy-directory-backed-struts-commerce/src/main/java/com/acme/legacy/billing/BillingRefundRepository.java
package com.acme.legacy.billing;

public class BillingRefundRepository {
  public RefundAuditResult writeRefundAuditEntry(String refundId) {
    String sql = "CALL rebuild_struts_refund_audit(); SELECT * FROM billing.struts_refund_audit_entries";
    return new RefundAuditResult(sql);
  }
}

File: eval/fixtures/legacy-directory-backed-webforms-commerce/Billing/BillingRefundRepository.cs
namespace Acme.Legacy.Billing;

public class BillingRefundRepository
{
  public string WriteRefundAuditEntry(string refundId)
  {
    string sql = "EXEC billing.rebuild_webforms_refund_audit; SELECT * FROM billing.webforms_refund_audit_entries";
    return sql;
  }
}
